test/test_pandas_pipelines.py

```diff
@@ -17,12 +17,12 @@ def is_equal_code(code1: str, code2: str) -> bool:
     join_pipeline_examples,
 )
 def test_join_pipeline(pipeline: PipelineExample):
-    assert is_equal_code(Orchestrator().transform(pipeline.code), pipeline.optimized_code)
+    assert Orchestrator().transform(pipeline.code).replace("\n", "") == pipeline.optimized_code.replace("\n", "")
```

Comment:
I actually liked that abstraction :D
If you don't want it anymore, could you also remove the function?

Reply:
You're right. I came with a solution that makes us both happy :D